src/main/java/cam72cam/immersiverailroading/model/ModelState.java

```diff
@@ -300,9 +301,13 @@ public void render(OBJRender.Binding vbo, EntityMoveableRollingStock stock, List
 
         levels.forEach((level, litGroups) -> {
             List<String> animated = animatedGroups.isEmpty() ? Collections.emptyList() :
-                    litGroups.stream().filter(g -> animatedGroups.containsKey(g)).collect(Collectors.toList());
-            List<String> notAnimated = animatedGroups.isEmpty() ? litGroups :
-                    litGroups.stream().filter(g -> !animatedGroups.containsKey(g)).collect(Collectors.toList());
+                    litGroups.stream().filter(g -> animatedGroups.containsKey(g)).filter(g -> !g.contains("ALPHA")).collect(Collectors.toList());
```

Comment:
All of these streams should be replaced with a single loop at this point for performance.